.eslintignore

```diff
@@ -1,24 +1,11 @@
-node_modules
+**/fixtures/**
+**/mock/store/**
+**/partials/**
+**/types/custom.d.ts
+build/
 coverage/
-wiki/
-static/
 flow-typed/
+node_modules/
+static/
 website/
-build/
-*.md
-*.lock
-*.yaml
-Dockerfile
-*.rpi
-*.html
-*.scss
-*.png
-*.json
-*.name
-*.tgz
-*.jpg
-*.sh
```

Comment:
we don't need to exclude all these files since they are not part of the input files